code/__DEFINES/exosuit_fab.dm

```diff
@@ -0,0 +1,34 @@
+/// Module is compatible with Security Cyborg models
+#define BORG_MODULE_SECURITY 		(1<<0)
+/// Module is compatible with Miner Cyborg models
+#define BORG_MODULE_MINER			(1<<1)
+/// Module is compatible with Janitor Cyborg models
+#define BORG_MODULE_JANITOR			(1<<2)
+/// Module is compatible with Medical Cyborg models
+#define BORG_MODULE_MEDICAL			(1<<3)
+/// Module is compatible with Engineering Cyborg models
+#define BORG_MODULE_ENGINEERING		(1<<4)
+/// Module is used for service Cyborgs specialization
+#define BORG_MODULE_SPECIALITY		(1<<5)
+
+/// Module is compatible with Ripley Exosuit models
+#define EXOSUIT_MODULE_RIPLEY		(1<<0)
+/// Module is compatible with Odyseeus Exosuit models
+#define EXOSUIT_MODULE_ODYSSEUS		(1<<1)
+/// Module is compatible with Clarke Exosuit models
+#define EXOSUIT_MODULE_CLARKE		(1<<2)
```

Comment:
We don't have the Clarke mech and the firefighter is missing

Reply:
Firefighter has the same module set as ripley